### ISSUE TITLE
http calls are erroring

### ISSUE THREAD

**Issue by marcesher** [statianzo/hubot-newrelic2#13]
Greetings. Thanks for creating this, @statianzo... looks really useful.

I'm trying to use the latest. I've configured as documented, but when I try running `hubot newrelic [any-command]` I get this:

```
hubot> hubot newrelic users
hubot> [Wed Apr 19 2017 20:18:54 GMT-0400 (EDT)] ERROR TypeError: Cannot read property 'on' of undefined
  at /Users/esherm/dev/projects/javascript/CFPBot/node_modules/scoped-http-client/src/index.js:87:14
  at get (/Users/esherm/dev/projects/javascript/CFPBot/node_modules/hubot-newrelic2/src/newrelic.coffee:77:44, <js>:50:34)
  at TextListener.callback (/Users/esherm/dev/projects/javascript/CFPBot/node_modules/hubot-newrelic2/src/newrelic.coffee:137:5, <js>:118:14)
```

I've tried with node versions 5, 6, and 7, and end up with the same error on all. Other installed scripts that use `robot.http` work just fine, so I'm not sure what to make of it failing in the newrelic script.

Any ideas?

Thanks!

**Comment by cmckni3** [comment]
I can look at it in a few minutes. What version of hubot are you using?

**Comment by marcesher** [comment]
Thanks @cmckni3 ! 

looks like 2.19.0, with version 0.11.0 of scoped-http-client

I wonder if it's related to conflicting versions of coffeescript.

**Comment by cmckni3** [comment]
Coffeescript shouldn't be an issue. The http client could be the culprit if it doesn't use callbacks anymore.

**Comment by cmckni3** [comment]
`master` works for me with `hubot@2.19.0` and `scoped-http-client@0.11.0` 🤔

**Comment by cmckni3** [comment]
Does your project have a version of coffeescript specified?

**Comment by marcesher** [comment]
hey @cmckni3 I just figured it out. I typo'd the api key name!  So the api key wasn't actually being set, and what was happening was that inside the http client, it was hitting a trapped error. Once I added enough logging into the client to see what was happening, the fix was easy... fix the darn typo.

I'm happy to submit a PR to validate the key is correctly set. If you have a preferred style for that, let me know.

Thanks again for helping me sort through this. I really appreciate your time.
